tests/Application/package.json.~1.11.0.dist

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
please add some info to the docs about that user have to manualy copy package.json~1.x.0.dist to package.json

eg. https://github.com/BitBagCommerce/SyliusWishlistPlugin/blob/master/doc/05-testing.md#testing